packages/terra-search-field/src/SearchField.jsx

```diff
@@ -220,10 +220,12 @@ class SearchField extends React.Component {
       customProps.className,
     ]);
 
-    const inputText = intl.formatMessage({ id: 'Terra.searchField.search' });
+    // eslint-disable-next-line no-prototype-builtins
+    const inputText = inputAttributes && inputAttributes.hasOwnProperty('aria-label') ? inputAttributes['aria-label'] : intl.formatMessage({ id: 'Terra.searchField.search' });
```

Comment:
Let's try this and remove the eslint-disable comment:
```suggestion
    const inputText = inputAttributes && Object.prototype.hasOwnProperty.call(inputAttributes, 'aria-label') ? inputAttributes['aria-label'] : intl.formatMessage({ id: 'Terra.searchField.search' });
```

Reply:
fixed here https://github.com/cerner/terra-core/pull/2587/commits/9ff4a13924754de046bea15ec89ec3056de74478.